docs/content/concepts/partitions-schedules-sensors/backfills.mdx

```diff
@@ -33,44 +33,36 @@ width={1737}
 height={335}
 />
 
-#### Single-run backfills
+#### Backfill policies and single-run backfills (Experimental)
 
 By default, if you launch a backfill that covers N partitions, Dagster will launch N separate runs - one for each partition. This works well when your code is single threaded, because it avoids overwhelming it with large amounts of data. However, if you're using a parallel-processing engine like Spark and Snowflake, you often don't need Dagster to help with parallelism, so splitting up the backfill into multiple runs just adds extra overhead.
 
 Dagster supports backfills that execute as a single run that covers a range of partitions. For example, this allows you to execute the backfill as a single Snowflake query. After it completes, Dagster will track that all those partitions have been filled.
 
-For this to work, your code needs to be written in a way that allows it to operate on partition ranges, instead of just single partitions.
+To get this behavior, you need to do two things:
 
-If your code uses any of the `partition_key`, `asset_partition_key`, `asset_partition_key_for_output`, or `asset_partition_key_for_input` context methods or properties, you'll need to update it to use methods or properties like `asset_partitions_time_window`, `asset_partition_key_range`, `asset_partition_keys`, `asset_partitions_time_window_for_output`, `asset_partition_key_range_output`, `asset_partitions_time_window_for_input`, or `asset_partition_key_range_for_input` instead.
+- Set the `backfill_policy` on your asset to <PyObject object="BackfillPolicy" method="single_run" />.
+- Write your code in a way that allows it to operate on partition ranges, instead of just single partitions. This means that, if your code uses the `partition_key` context property, you'll need to update it to instead use one of the `partition_time_window`, `partition_key_range`, or `partition_keys` properties.
```

Comment:
Suggestion: some users have reported confusion around this particular line in the docs.

Perhaps overcommunicating, but maybe we can say: 
```suggestion
- Write your code in a way that allows it to operate on partition ranges, instead of just single partitions. This means that, if your code uses the `partition_key` context property, you'll need to update it to instead use the `partition_keys` property. Then, you can run your code on each executing partition key.

For time window partitions definitions, you can use the `partition_time_window` context property to fetch the time window of the executing partitions. Or, for other partitions definitions, you can use `partition_key_range` to fetch the start and end partitions in the targeted range.
```

Reply:
I pushed an update that adds some clarification. Let me know what you think.